spec/lib/jwt_helper_spec.rb

```diff
@@ -5,10 +5,13 @@
     it "appends a valid JWT token in the querystring" do
       url = "http://step-nav-slug"
 
-      expected_token = "eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOjEyM30.-r0B06VSBvNo2Z_WMSuC9G65epKc3ZdYxM8L-3xfsvI"
-      expected_url = "#{url}?token=#{expected_token}"
-
-      expect(described_class.access_limited_preview_url(url, 123)).to eq(expected_url)
+      # The generated token value now changes every second due to the time-related `iat` and `exp` fields,
+      # so we can only consistently reproduce the token value by setting the system time.
+      Timecop.travel Time.zone.local(2020, 1, 1, 10, 0, 0) do
+        expected_token = "eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOjEyMywiaXNzIjo5LCJpYXQiOjE1Nzc4NzI4MDAsImV4cCI6MTU4MDU1MTIwMCwiZHJhZnRfYXNzZXRfbWFuYWdlcl9hY2Nlc3MiOnRydWUsImNvbnRlbnRfaWQiOjd9.QDj7svewjQcLhikv_o37cy7_dBPH-ME2e-0ec5IcIn4"
+        expected_url = "#{url}?token=#{expected_token}"
+        expect(described_class.access_limited_preview_url(url, 123, 7, 9)).to eq(expected_url)
+      end
```

Comment:
Talking of the JwtHelper smell it feels a bit strange that we'd need essentially the same test here and in application helper.

Reply:
Have removed JwtHelper in my latest version of the branch.